distributed-tracing/node-js-app/README.md

```diff
@@ -0,0 +1,43 @@
+# Wavefront Hackathon - NodeJS App
+
+This is a sample NodeJS application called beachshirts (#[beachops](https://medium.com/@matthewzeier/thoughts-from-an-operations-wrangler-how-we-use-alerts-to-monitor-wavefront-71329c5e57a8)) which makes cool shirts for the beach.
+
+### Running the Application Locally
+1. Run Jaeger in your env using the [Docker image](https://www.jaegertracing.io/docs/getting-started):
+    ```
+       $ docker run -d --name jaeger \
+         -e COLLECTOR_ZIPKIN_HTTP_PORT=9411 \
+         -p 5775:5775/udp \
+         -p 6831:6831/udp \
+         -p 6832:6832/udp \
+         -p 5778:5778 \
+         -p 16686:16686 \
+         -p 14268:14268 \
+         -p 9411:9411 \
+         jaegertracing/all-in-one:1.8
+    ```
+
+2. git clone this repo and navigate to this dir:
+
+    ```
+     git clone https://github.com/wavefrontHQ/hackathon.git 
+     cd hackathon/distributed-tracing/node-js-app
+    ```
+
+3. Install the dependencies using below commands:
+    ```
+    npm install
+    ```
+
+4. Start the services using below commands:
+    ```
+    node beachshirt/app.js
```

Comment:
Great document, I would suggest pointing out minimal NodeJs version required.  We don't want new users to run around troubleshooting npm install or node runtime errors.